install-dev/data/xml/feature_flag.xml

```diff
@@ -13,5 +13,6 @@
     <feature_flag id="product_page_v2" name="product_page_v2" label_wording="New product page - Single store" label_domain="Admin.Advparameters.Feature" description_wording="This page benefits from increased performance and includes new features such as a new combination management system." description_domain="Admin.Advparameters.Help" state="0" stability="beta" />
     <feature_flag id="product_page_v2_multi_shop" name="product_page_v2_multi_shop" label_wording="New product page - Multi store" label_domain="Admin.Advparameters.Feature" description_wording="Access the new product page, even in a multistore context. This is a work in progress and some features are not available." description_domain="Admin.Advparameters.Help" state="0" stability="beta" />
     <feature_flag id="attribute_group" name="attribute_group" label_wording="Attribute group" label_domain="Admin.Advparameters.Feature" description_wording="Enable / Disable migrated attribute group page." description_domain="Admin.Advparameters.Help" state="0" stability="beta" />
+    <feature_flag id="multiple_image_format" name="multiple_image_format" label_wording="Multiple image formats" label_domain="Admin.Advparameters.Feature" description_wording="Enable / Disable having more than one image format (jpg, webp, avif, png...)" description_domain="Admin.Advparameters.Help" state="0" stability="beta" />
```

Comment:
You must add these new wordings in the [FeatureFlagLang](https://github.com/PrestaShop/PrestaShop/blob/82be101a05cdd528ccc8f9ffc50742bb24fcf3d4/classes/lang/KeysReference/FeatureFlagLang.php#L45-L44) file so that that can scanned and integrated in the catalog later

https://github.com/PrestaShop/PrestaShop/blob/82be101a05cdd528ccc8f9ffc50742bb24fcf3d4/classes/lang/KeysReference/FeatureFlagLang.php#L45-L44